internal/bminventory/inventory.go

```diff
@@ -1996,7 +1999,32 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 	return nil
 }
 
-func validateUserManagedNetworkConflicts(params *models.ClusterUpdateParams, log logrus.FieldLogger) error {
+func setCommonUserNetworkManagedParams(params *models.ClusterUpdateParams, cluster *common.Cluster, updates map[string]interface{}, log logrus.FieldLogger) error {
+	err := validateUserManagedNetworkConflicts(params, cluster, log)
+	if err != nil {
+		return err
+	}
+	updates["vip_dhcp_allocation"] = false
+	updates["api_vip"] = ""
+	updates["ingress_vip"] = ""
+	return nil
+}
+
+func updateNtpSources(params installer.UpdateClusterParams, updates map[string]interface{}, log logrus.FieldLogger) error {
+	if params.ClusterUpdateParams.AdditionalNtpSource != nil {
+		ntpSource := swag.StringValue(params.ClusterUpdateParams.AdditionalNtpSource)
+
+		if ntpSource != "" && !validations.ValidateAdditionalNTPSource(ntpSource) {
+			err := errors.Errorf("Invalid NTP source: %s", ntpSource)
+			log.WithError(err)
+			return common.NewApiError(http.StatusBadRequest, err)
+		}
+		updates["additional_ntp_source"] = ntpSource
+	}
+	return nil
+}
+
+func validateUserManagedNetworkConflicts(params *models.ClusterUpdateParams, cluster *common.Cluster, log logrus.FieldLogger) error {
```

Comment:
Seems that this method doesn't require `*common.Cluster` it just needs to know if the cluster is single node or not.

Reply:
ok, sounds better, agree